.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
 language: python
 
 python:
-  - "3.6"
+  - "3.7"
```

Comment:
To use >=3.7 you need to add `dist: xenial`, see https://docs.travis-ci.com/user/languages/python/